scss/_base_typography.scss

```diff
@@ -40,6 +40,14 @@
     }
   }
 
+  @if ($increase-font-size-on-larger-screens) {
+    @media screen and (min-width: $breakpoint-x-large) {
+      *:not(html) {
+        line-height: map-get($line-heights, default-text);
```

Comment:
Will this not set all line-heights to the same value on desktop screens? Don't we want headings for example to have more line height?

Reply:
@anthonydillon Things that require a different line-height have that already set with a higher specificity rule. This is meant to handle elements that inherit from html. Percy hasn't generated a diff, so I think it's safe to say there there haven't been any side effects.

This just gets around the fact that setting 
```
html {
    font-size: 1.125rem; 
    line-height: 1.5rem
}
``` 
results in two things:
• for the html element, line-height iscomputed as if the font-size hasn't changed, so 24px (16 * 1.5)
• anything that inherits line-height from html however uses the font-size increase, so its line-height becomes 1.125rem * 1.5.

This begs the question why are we setting line-height as 1.125rem instead of 18px, can't remember off the top of my head, but there it was causing issues. I can look into it and provide a clearer answer if needed.

So all this pr does is set line-height for things that inherit from html to the expected value, 1.5rem.

There might be a better way to do this, but I haven't found one so far.

Reply:
@anthonydillon this is so we don't have to put explicit rules on things like .p-navigation__link etc. they have no line-height specifically set and become taller than they should be on screens > 1680px.